workshops/dodge/README.md

```diff
@@ -4,7 +4,7 @@ Short link to this workshop: https://workshops.hackclub.com/dodge
 
 ---
 
-In this workshop we're going to build a simple yet addictive game using JavaScript. Previously when we've done web development projects, we've used HTML elements and things of that ilk to build what we see in the browser. Today we're going to do things a little differently and use some new tools-- JavaScript, along with two great libraries called p5.js and p5.play, to build our game.
+In this workshop, we're going to build a simple, yet addictive game using JavaScript. Previously when we've done web development, we've used HTML elements and things of that ilk to build what we see in the browser. Today we're going to do things a little differently and use a new coding language, JavaScript, along with two great libraries called p5.js and p5.play, to build our game.
```

Comment:
Ah, good catch. I did that in the changes I was making on our call
yesterday, but later reverted it for unknown reasons.

On Wed, Apr 6, 2016 at 5:43 PM, JESSICA <notifications@github.com> wrote:

> In workshops/dodge/README.md
> <https://github.com/jkwok91/hackclub/pull/3#discussion_r58805132>:
>
> > @@ -4,7 +4,7 @@ Short link to this workshop: https://workshops.hackclub.com/dodge
> >
> >  ---
> >
> > -In this workshop we're going to build a simple yet addictive game using JavaScript. Previously when we've done web development projects, we've used HTML elements and things of that ilk to build what we see in the browser. Today we're going to do things a little differently and use some new tools-- JavaScript, along with two great libraries called p5.js and p5.play, to build our game.
> > +In this workshop, we're going to build a simple, yet addictive game using JavaScript. Previously when we've done web development, we've used HTML elements and things of that ilk to build what we see in the browser. Today we're going to do things a little differently and use a new coding language, JavaScript, along with two great libraries called p5.js and p5.play, to build our game.
>
> ey yo if you're gonna get down with commas you gotta put one after "yet
> addictive" too
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jkwok91/hackclub/pull/3/files/9f22de72a8edfb25fab672cded6e6bf57e8ffcbd#r58805132>
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>